src/AutoRest.CSharp/Mgmt/Decorator/MethodExtensions.cs

```diff
@@ -46,10 +46,9 @@ internal static bool IsListMethod(this RestClientMethod method, [MaybeNullWhen(f
             }
             else
             {
-                string pagingItemName = method.Operation.Paging?.ItemName ?? "value";
+                valuePropertyName = method.Operation.Paging?.ItemName ?? "value";
                 var schemaObject = (SchemaObjectType)returnType.Implementation;
-                itemType = GetValueProperty(schemaObject, pagingItemName)?.ValueType.Arguments.FirstOrDefault();
-                valuePropertyName = pagingItemName.ToCleanName();
```

Comment:
This `ToCleanName` call has been causing the issue when `"Value"` item property name is generated where `"value"` is expected

Reply:
Previously this depends on the paged list result models in which has a property `Value` therefore we called `ToCleanName` here. Since our new helpers will parse the raw json to get the value of array, we no longer need this.